core/shared/src/main/scala/zio/internal/FiberContext.scala

```diff
@@ -128,10 +128,30 @@ private[zio] final class FiberContext[E, A](
   private[this] final def cutAncestryTrace(trace: ZTrace): ZTrace = {
     val maxExecLength  = platform.tracing.tracingConfig.ancestorExecutionTraceLength
     val maxStackLength = platform.tracing.tracingConfig.ancestorStackTraceLength
+    val maxAncestors   = platform.tracing.tracingConfig.ancestryLength - 1
+
+    val flatAncestors = {
```

Comment:
Can we push this out into a helper method, maybe in `ZTrace` companion object?

Reply:
Replaced by a call to `trace.parents`